fhireval/data/BasicTest.json

```diff
@@ -0,0 +1,328 @@
+{
+	"groupname": "BasicTest",
+	"datatype": "TestOnly",
+	"resources": {
+		"Patient" : [
+			{
+				"resourceType": "Patient",
+				"extension": [
+				  {
+					"url": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-race",
+					"extension": [
+					  {
+						"url": "ombCategory",
+						"valueCoding": {
+						  "system": "urn:oid:2.16.840.1.113883.6.238",
+						  "code": "2028-9",
+						  "display": "Asian"
+						}
+					  },
+					  {
+						"url": "text",
+						"valueString": "Asian:Middle Eastern (from the United Arab Emirates)"
+					  }
+					]
+				  }, {
+					"url": "https://ncpi-fhir.github.io/StructureDefinition/visit-number",
```

Comment:
What is this extension?

Reply:
It's an extension used for testing. That extension is defined inside that same file, but toward the end of the file.